examples/simple-router/simple-router.rb

```diff
@@ -0,0 +1,167 @@
+require "pio"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.